packages/cli/src/utils/nps/capture.ts

```diff
@@ -0,0 +1,43 @@
+import fetch from 'node-fetch'
+
+export interface EventCapture {
+  capture<Event>(id: string, name: string, payload: Event): Promise<void>
```

Comment:
In TypeScript, unlike other languages, generics are not so much an instrument of polymorphism but rather of propagating type constraints; polymorphism is primarily achieved through subtyping. A lone type variable that is not mentioned in the signature again is equivalent to just specifying its bound literally, or, in the case of an unconstrained type variable, the top type:
```suggestion
  capture(id: string, name: string, payload: unknown): Promise<void>
```

Reply:
done